README.md

```diff
@@ -1,68 +1,132 @@
 
-## 💎 Core concepts
+**Developer-Friendly**
 
-- All in one place: here, you can find everything you need to build a modern and reliable web application with Uploadcare on board.
-- Modern technologies: web components, service workers, wasm, and other cutting edge web standards are ready to show their power.
-- Type safety with a JSDoc declaration and TypeScript static analysis.
-- Modern ESM-level code sharing.
-- The build stage is unnecessary for development: use the code directly in any modern browser or Node.js.
-- Minimum of external dependencies: security audit is not expensive.
-- Fast and secure.
-- We try to be as close to the native web platform as possible and not force you to use any specific development environment tools.
+`Uploadcare Blocks` come with modern technologies at your fingertips, like [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components), [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API), [WASM](https://webassembly.org/), [ESM-level](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) code sharing, and other cutting-edge web standards. `Uploadcare Blocks` is a native web platform with minimum external dependencies and a lightweight library providing a cheap security audit.
 
-## 📦 What’s in the box?
+**Typescript support**
 
-- [File uploading solutions](/solutions/file-uploader/) — file uploader implementations for the most frequent use cases.
-- [Blocks to build custom solutions](/blocks/) — build your integrations with a simple HTML and CSS.
-- [Adaptive image](/solutions/adaptive-image/) — all you need to distribute your web images efficiently.
-- [Cloud image editor](/solutions/cloud-image-editor/) — tune your pictures with our CDN image processing right after they're uploaded.
-- Soon: web components for media content representation: audio, video, image galleries, etc.
+We use [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for TypeScript static analysis support during development.  Additionally, we provide type definitions ([\*.d.ts files](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html)) for the TypeScript projects in our packages. Check the [JSDoc Reference](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) page in TypeScript official documentation.
 
-## ⚙️ Installation and start
+## 🚀 Getting Started
 
-Installation process is easy and takes minutes. See the [installation guide](/get-started/installation/).
+### HTML `<script>` Tag
 
-## Typescript support
+* Connect `Blocks` directly from your document:
 
-We use [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for TypeScript static analysis support during development.
+```html
+<script src="https://unpkg.com/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/blocks-browser.min.js" type="module"></script>
```

Comment:
```suggestion
<script type="module">
  import * as LR from 'https://unpkg.com/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/blocks.min.js';
  LR.registerBlocks(LR);
</script>
```

Reply:
And also there is no `{{PACKAGE_VERSION}}` auto-injection. I think we should explicitly say that user need to replace it manually, or just omit package version at all. First option is better I think.